### PR TITLE
Wazuh indexer 4.4.0 with Openseach 2.4.1 to 4.4

### PR DESCRIPTION
**Pull request by c-bordon**
|Related issue|
|---|
|https://github.com/wazuh/wazuh-jenkins/issues/4508|

## Description

The same changes were applied for the 4.3.11 package and can be seen in this PR
https://github.com/wazuh/wazuh-packages/pull/1967
## Tests

- Build the package in any supported platform
  - [x] Linux
  - [ ] Windows
  - [ ] macOS
  - [ ] Solaris
  - [ ] AIX
  - [ ] HP-UX
- [x] Package installation
- [x] Package upgrade
- [ ] Package downgrade
- [x] Package remove
- [ ] Package install/remove/install
- [ ] Change added to CHANGELOG.md

- Tests for Linux RPM
  - [x] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] `%files` section is correctly updated if necessary
- Tests for Linux deb
  - [x] Build the package for x86_64
  - [ ] Build the package for i386
  - [ ] Build the package for armhf
  - [ ] Build the package for aarch64
  - [ ] Package install/remove/install
  - [ ] Package install/purge/install
  - [ ] Check file permissions after installing the package
- Tests for macOS
  - [ ] Test the package from macOS Sierra to Mojave
- Tests for Solaris
  - [ ] Test the package on Solaris 10
  - [ ] Test the package on Solaris 11
  - [ ] Check file permissions on Solaris 11 template
- Tests for IBM AIX
  - [ ] `%files` section is correctly updated if necessary
  - [ ] Check the changes from IBM AIX 5 to 7

### CODE REVIEW

**Review by okynos — CHANGES_REQUESTED** [review]
Remove commented files in SPECs. GJ

**Comment by c-bordon** [comment]
> Remove commented files in SPECs. GJ

Done: 77a76e12

**Comment by c-bordon** [comment]
A new set of tests for 2.4.1: https://ci.wazuh.info/view/Tests/job/Test_stack_tier/8/console

**Review by okynos — APPROVED** [review]

File: stack/indexer/base/builder.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -x
```
Comment (verdx) [inline stack/indexer/base/builder.sh:3]: This should be removed as a debug element

File: alpine/SPECS/wazuh-agent/APKBUILD [hunk 2]
```diff
@@ -7,7 +7,7 @@ pkgdesc="Wazuh helps you to gain security visibility into your infrastructure."
 url="https://www.wazuh.com/"
 arch="${architecture}"
 license="GPL"
-depends="bash busybox procps"
+depends="busybox procps"
```
Comment (verdx) [inline alpine/SPECS/wazuh-agent/APKBUILD:10]: Why is `bash` removed as a dependency?

File: stack/indexer/rpm/build_package.sh [hunk 3]
```diff
@@ -8,7 +8,7 @@
 # License (version 2) as published by the FSF - Free Software
 # Foundation.
 
-set -e
+set -ex
```
Comment (verdx) [inline stack/indexer/rpm/build_package.sh:11]: Debug element?

File: stack/indexer/rpm/builder.sh [hunk 4]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -x
```
Comment (verdx) [inline stack/indexer/rpm/builder.sh:3]: Debug element?

File: alpine/SPECS/wazuh-agent/APKBUILD (hunk 2 above)
Comment (okynos) [reply alpine/SPECS/wazuh-agent/APKBUILD:10]: The scripts work with SH and not even need bash.

File: stack/indexer/base/builder.sh (hunk 1 above)
Comment (okynos) [reply stack/indexer/base/builder.sh:3]: It's good to include build information, in builders doesn't mind the additional amount of information,

File: stack/indexer/rpm/build_package.sh (hunk 3 above)
Comment (okynos) [reply stack/indexer/rpm/build_package.sh:11]: Additional information generator, I think it's required.

File: stack/indexer/rpm/builder.sh (hunk 4 above)
Comment (okynos) [reply stack/indexer/rpm/builder.sh:3]: Additional information generator, I think it's required.

**Review by okynos — APPROVED** [review]

**Review by verdx — APPROVED** [review]
LGTM

**Pull request merged**